src/fsharp/NicePrint.fs

```diff
@@ -1405,12 +1405,24 @@ module InfoMemberPrinting =
             | None -> tagProperty
             | Some vref -> tagProperty >> mkNav vref.DefinitionRange
         let nameL = DemangleOperatorNameAsLayout tagProp pinfo.PropertyName
+        let getterSetter =
+            match pinfo.HasGetter, pinfo.HasSetter with
+            | (true, false) ->
+                "with get" |> tagText |> wordL 
```

Comment:
No longer highlighting the `with` as a keyword here?

Reply:
Whoops, my bad. Will fix.

Reply:
Done.